src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/UnitDeltaSockDTO.java

```diff
@@ -0,0 +1,22 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto;
+
+public class UnitDeltaSockDTO {
```

Comment:
UnitDeltaSock? :D

<img width="805" alt="Bildschirmfoto 2022-05-08 um 12 32 53" src="https://user-images.githubusercontent.com/91155454/167292184-820962a5-eb19-4eb4-bc59-f442ae3de177.png">

What's the purpose of this class? We need it when for example a unit was killed and we want to tell the client about that over WebSocket, right?

Reply:
:D

Reply:
![delta](https://user-images.githubusercontent.com/91190011/168173018-2a5ee1f7-40d3-447e-bff3-aa0505e0297e.png)